maintainers/backporting.md

```diff
@@ -0,0 +1,8 @@
+
+# Backporting
+
+Backports are handled by the backport action.
+
+Since GitHub Actions can not trigger actions, the backport PR needs to be re-triggered by another actor. This is achieved by closing and reopening the backport PR.
```

Comment:
```suggestion
Since [GitHub Actions workflows will not trigger other workflows](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow), checks on the automatic backport need to be triggered by another actor.
This is achieved by closing and reopening the backport pull request.
```